src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -371,6 +372,9 @@ static gyroSpiDetectFn_t gyroSpiDetectFnTable[] = {
 #ifdef USE_GYRO_SPI_ICM20649
     icm20649SpiDetect,
 #endif
+#if defined(USE_GYRO_SPI_ICM45686)
```

Comment:
Would prefer (as used in the config)
```suggestion
#if defined(USE_ACCGYRO_ICM45686)
```